src/components/State/Store.tsx

```diff
@@ -47,6 +47,15 @@ type Store = {
     showHiddenTags: boolean;
     setShowHiddenTags: (show: boolean) => void;
 
+    tags: { tagId: string, tagName: string, newValue: string }[];
+    setTags: (tags: { tagId: string, tagName: string, newValue: string }[]) => void;
+
+    formattedData: any[];
```

Comment:
```suggestion
    formattedData: <type>;
```
find the type or create a new one, avoid any